web/src/app/alerts/components/AlertDetails.tsx

```diff
@@ -208,10 +240,10 @@ export default function AlertDetails(props) {
     return 'None'
   }
 
-  function renderEscalationPolicySteps() {
+  function renderEscalationPolicySteps(): ReactNode {
```

Comment:
I think this one can be `JSX.Element`. I generally use that type when returning components, and I use the `ReactNode` return type if there's a possibility that it'll return null